docs/UserGuide.adoc

```diff
@@ -545,6 +545,23 @@ Example:
 * `done icP/S7896432A icD/S4578902G res/fail`
 // end::mark[]
 
+// tag::delete[]
+=== Deleting a person : `delete`
+
+If you want to delete a person in ORGANice you can type in a command in the following format: `delete **NRIC**`  +
+ORGANice will prevent you from deleting persons in the following categories:
```

Comment:
ORGANice will prevent you from deleting a person if he/she is any of the following:

* A patient whose status is currently under processing.
* A donor whose status is currently under processing.
* A doctor who is in charge of one or more patients in ORGANice.